collection/rancher/v2.x/profile-collector/continuous_profiling.sh

```diff
@@ -188,14 +189,31 @@ while getopts "a:p:d:s:t:h" opt; do
   esac
 done
 
-timestamp() {
-  date "+%Y-%m-%d %H:%M:%S"
+loglevel() {
+  kubectl -n $NAMESPACE get pods -l app=$APP --no-headers -o custom-columns=name:.metadata.name | while read rancherpod; do
+    techo Setting $rancherpod $1 logging
+    kubectl -n $NAMESPACE exec $rancherpod -c rancher -- loglevel --set $1
+  done
 
 }
 
-techo() {
-  echo "$(timestamp): $*"
-
-}
+# set the container for the app being profiled
+case $APP in
+rancher)
+  CONTAINER=rancher
+  loglevel debug
```

Comment:
This is a bit surprising here. I suggest to move that to the top of `collect` - this chunk of code is only expected to set CONTAINER and NAMESPACE

Reply:
Move  to the top of collect, before the loop